Gemfile

```diff
@@ -75,6 +75,9 @@ gem 'rubocop', '>= 1.0', '< 2.0'
 
 gem 'rspec', '>=3.0', '< 4.0'
 
+# Run against this stable release
 group :development, :test do
-  gem 'rspec-rails'
+  gem 'rspec-rails', '~> 6.0.0'
```

Comment:
## This inline comment is for README file, but I will added here since no change was made there and I cannot select it:
- [ ] Please add the necessary commands that the user must do to run your tests. Also at your commands kindly include/explain to potential users how to create/migrate your db